test/Microsoft.TestPlatform.AcceptanceTests/AcceptanceTestBase.cs

```diff
@@ -9,13 +9,32 @@ namespace Microsoft.TestPlatform.AcceptanceTests
 
     public class AcceptanceTestBase : IntegrationTestBase
     {
+        public const string Net451TargetFramework = "net451";
+        public const string Net452TargetFramework = "net452";
+        public const string Net46TargetFramework = "net46";
+        public const string Net461TargetFramework = "net461";
+        public const string Net462TargetFramework = "net462";
+        public const string Net47TargetFramework = "net47";
+        public const string Net471TargetFramework = "net471";
+        public const string Net472TargetFramework = "net472";
+        public const string Net48TargetFramework = "net48";
         public const string DesktopTargetFramework = "net451";
```

Comment:
Should we use `Net451TargetFramework` instead of this?

Reply:
I would rather avoid mixing those changes into this PR, because the DesktopTargetFramework is used in multiple places and in conjunction with IsDesktopRunner, which would prompt even more changes to make the code sense semantically. 

![image](https://user-images.githubusercontent.com/5735905/88772900-b66ffb80-d181-11ea-82a3-b8b21edb5ea7.png)

I am sure I will come across this again when I work with linux tests and expanding our testing to net5.0, so I can fix it then.